hachoir/parser/program/macho.py

```diff
@@ -266,6 +394,10 @@ def createFields(self):
         desc, parser = self.LOAD_COMMANDS.get(self['cmd'].value, ("", None))
         if parser:
             yield parser(self, "data")
+            # data is word aligned. TODO: 8 bytes for 64-bit
```

Comment:
Instead of trying to align to 64-bits, can you instead insert padding according to `cmdsize`?